models/notification.model.js

```
@@ -0,0 +1,39 @@
+    default: false,
+  },
+  createdAt: {
+    type: Date,
+    default: Date.now,
+  },
+}, {
+  timestamps: true,
+});
+
+
+export default Notification = mongoose.model("Notification", NotificationSchema);
```

Biome (1.9.4)

[error] A global variable should not be reassigned.

Assigning to a global variable can override essential functionality.

(lint/suspicious/noGlobalAssign)